domains/sycured.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "sycured",
+        "email": "sycured@protonmail.com"
+    },
+    "record": {
+        "URL": "https://sycured.com"
```

Comment:
We are currently working on a bug related to url redirections #221. This may take a week to fix. I'd suggest using a cname record instead. If you want to keep it as it is, let me know, I'll merge it in.

Reply:
I want to keep it as it is. Thanks

Reply:
Alright. You can subscribe to #221 to get updates on the issue.